recipe/meta.yaml

```diff
@@ -91,28 +104,52 @@ outputs:
         - cuda-version >=11.0.*,<12.0.a0  # [(cuda_compiler_version or "").startswith("11")]
       run_constrained:
         - cuda-version >=11.0.*,<13.a0    # [cuda_compiler_version == "None"]
-        - mkl-devel 2024.4
+        - mkl >=2024
         - libnvpl-fft0 >=0.3.0,<1.0a0
+        - mathdx 24.04.0.*
+        - numba 0.60.*
+        # run_constrained because this is an off-channel package
+        - pynvjitlink >=0.2
     test:
+{% if is_gpu_available %}
+      source_files:
+        - examples/fft
+        - examples/linalg
+        - tests/example_tests
+{% endif %}
+      requires:
+        - cupy  # [cuda_compiler_version != "None"]
+        - pip
+        - pytest  # [cuda_compiler_version != "None"]
+        - pytorch  # [(cuda_compiler_version != "None") and (not win)]
       imports:
         - nvmath
+        - nvmath.fft
+        - nvmath.linalg
+      commands:
+        - pip check
+{% if is_gpu_available %}
+        - pytest tests/example_tests/matmul_tests/ -k "not cpu" -v -ra  # [cuda_compiler_version != "None"]
+        - pytest tests/example_tests/fft_tests/ -k "not cpu" -v -ra  # [cuda_compiler_version != "None"]
+{% endif %}
```

Comment:
Can we check 'pytest -v tests/nvmath_tests/fft tests/nvmath_tests/linalg' when gpu is not available?

Reply:
No. This version of nvmath-python does not offer a cpu-based backend for matmul.